src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -98,4 +99,11 @@ public String getServiceFileImportFromService(Interface service) {
   public String getProtoFileImportFromService(Interface service) {
     return service.getFile().getSimpleName().replace(".proto", "_pb");
   }
+
+  // For some reason this is only being called for tests.
+  @Override
+  public String getSourceFilePath(String path, String className) {
+    // Class name is some Java thing like LanguageServiceTest
+    return path + File.separator + className + "_test.jorge.rb";
```

Comment:
I think the following is better:

  public String getSourceFilePath(String path, String publicClassName) {
    return path + File.separator + Name.upperCamel(publicClassName).toLowerUnderscore() + ".rb";
  }